traffic_router/ultimate-test-harness/http_test.go

```diff
@@ -0,0 +1,497 @@
+package main
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"text/tabwriter"
+	"time"
+
+	go_log "github.com/apache/trafficcontrol/lib/go-log"
```

Comment:
why the alias?

Reply:
Because `"log"` is already imported. The alternative would be using `"github.com/apache/trafficcontrol/lib/go-log"` where `"log"` is currently used.

Reply:
Using `"github.com/apache/trafficcontrol/lib/go-log"` only in c593fc6450

Reply:
Oh, I thought I checked and didn't see "log" already used -- my bad